### PR TITLE
feat: unverified email page

### PR DESCRIPTION
**Pull request by danielgent**
Looks a little better on login2 as we're not using a generic page template

I'm not sure if anyone has actually confirmed this page but I don't imagine I came up with the styling myself :thinking:

### CODE REVIEW

File: src/routes/tsx/routes.tsx [hunk 1]
```diff
@@ -322,11 +323,7 @@ export const loginRoutes = new OpenAPIHono<{ Bindings: Env }>()
         // TODO! should do same as on ticket flow - send another validation email...
         // can then test for this
         return ctx.html(
-          <MessagePage
-            message={i18next.t("sent_code_spam")}
-            pageTitle={i18next.t("unverified_email")}
-            vendorSettings={vendorSettings}
```
Comment (danielgent) [inline src/routes/tsx/routes.tsx:328]: on login2 the styling is a little different as it's got an icon and some other custom code

File: src/utils/components/UnverifiedEmailPage.tsx
Comment (danielgent) [inline src/utils/components/UnverifiedEmailPage.tsx:1]: copy-pasted from login2

Comment (danielgent) [inline src/utils/components/UnverifiedEmailPage.tsx:1]: copy-pasted from login2 mostly

File: test/integration/login/__image_snapshots__/login-password-spec-ts-test-integration-login-login-password-spec-ts-register-password-should-create-a-new-user-with-a-password-and-only-allow-login-after-email-validation-2-snap.png
Comment (danielgent) [inline test/integration/login/__image_snapshots__/login-password-spec-ts-test-integration-login-login-password-spec-ts-register-password-should-create-a-new-user-with-a-password-and-only-allow-login-after-email-validation-2-snap.png:1]: voila

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**